src/Strategy/Node.hs

```diff
@@ -291,8 +311,36 @@ data NodeProject
   | NPM PkgJsonGraph
   deriving (Eq, Ord, Show, Generic)
 
+instance LicenseAnalyzeProject NodeProject where
+  licenseAnalyzeProject = pure . analyzeLicenses . pkgGraph
+
+analyzeLicenses :: PkgJsonGraph -> [LicenseResult]
+analyzeLicenses (PkgJsonGraph _ graph) = mapMaybe (uncurry mkLicenseResult) . toList $ graph
+
+mkLicenseResult :: Manifest -> PackageJson -> Maybe LicenseResult
+mkLicenseResult manifest PackageJson{..} = constrLicenseResult <$> allLicenses
+  where
+    manifestPath = toFilePath . unManifest $ manifest
+    allLicenses =
+      (singleton <$> packageLicense)
+        <> (map LicenseObj <$> packageLicenses)
+
+    mkLicense (LicenseText txt) = License UnknownType txt
+    mkLicense (LicenseObj pjlo) = License LicenseURL (licenseUrl pjlo)
+
+    constrLicenseResult licenses =
```

Comment:
Prefer: `mkLicenseResult`

Reply:
I tried that, but that's the same name as the upper level binding the where block is in, which is confusing. I think I'll leave it how it is for now.